src/types/Error.ts

```diff
@@ -0,0 +1,12 @@
+export enum Error {
```

Comment:
Consider renaming 'Error' to 'ErrorMessage' to avoid confusion with the built-in 'Error' class.